anvil/anvil.go

```diff
@@ -49,12 +56,23 @@ func (a *Anvil) Start(ctx context.Context) error {
 	anvilLog := a.log.New("chain.id", a.cfg.ChainId)
 	anvilLog.Info("starting anvil")
 
+	tempFile, err := os.CreateTemp("", "genesis-*.json")
```

Comment:
maybe adding a "./" or something would make it more clear, i think it's fine for now to just dump these to the root of our cwd, but we should figure out a single place to add any files that we generated

Reply:
this will add the file to a temp directory. Here is an example temp file that was created: `Temp file: /var/folders/0s/xwxh6fc52tq3fxyjvcl09_2c0000gn/T/genesis-1143869114.json`